docker.mk

```diff
@@ -11,7 +11,7 @@ VERSION="v$(MAJOR).$(MINOR).$(PATCH).$(BUILD)$(TYPE)"
 REGISTRY="localhost:5000/csi-powermax"
 
 docker:
-	docker build -t "$(REGISTRY):$(VERSION)" .
+	test -z "$(XGOARCH)" && docker build -t "$(REGISTRY):$(VERSION)" . || buildah bud -f Dockerfile.$(XGOARCH) -t $(REGISTRY):$(VERSION)
```

Comment:
the buildah is a binary to build OCI compatible images?
if you could declare buildah at the beginning of docker.mk and use it here as a variable it would be very helpful to understand better and some comment along with it.. It should be very helpful.
THanks

Reply:
Sorry, maybe I should have described it better, but yes, `buildah` is a widely used tool to build OCI compatible images. See: https://buildah.io/. I installed the `buildah` package through standard CentOS YUM repository on my system, just like `docker` command. How about adding a "pre-req" blurb in https://github.com/dell/csi-powermax#building?